app/assets/javascripts/components/select-with-search.js

```diff
@@ -24,7 +26,15 @@ window.GOVUK.Modules = window.GOVUK.Modules || {}
       shouldSort: false, // show options and groups in the order they were given
       itemSelectText: '',
       searchResultLimit: 100,
+      removeItemButton: this.select.multiple,
       labelId: this.select.id + '_label',
+      callbackOnInit: function () {
+        if (this.dropdown.type === 'select-multiple') {
```

Comment:
An inline comment explaining what this is doing / why this is needed would be good.

Reply:
OK done.